src/main/webapp/app/shared/common/edit-remove-icons/edit-remove-icons.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'jhi-edit-remove-icons',
```

Comment:
Could you rename it to something like `edit-remove-icons' instead of using prefix 'jhi' because this prefix means it's a built-in component of JHipster which is not correct in your case.